examples/notebooks/beam-ml/automatic_model_refresh.ipynb

```diff
@@ -57,7 +47,16 @@
     {
       "cell_type": "markdown",
       "source": [
-        "# Update ML models in running pipelines"
+        "# Update ML models in running pipelines\n",
+        "\n",
+        "<table align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/AnandInguva/beam/blob/notebook/beam/examples/notebooks/beam-ml/side_Input_model_updates.ipynb\"><img src=\"https://raw.githubusercontent.com/google/or-tools/main/tools/colab_32px.png\" />Run in Google Colab</a>\n",
```

Comment:
```suggestion
        "    <a target=\"_blank\" href=\"https://colab.sandbox.google.com/github/apache/beam/blob/master/examples/notebooks/beam-ml/automatic_model_refresh.ipynb\"><img src=\"https://raw.githubusercontent.com/google/or-tools/main/tools/colab_32px.png\" />Run in Google Colab</a>\n",
```

Right now this is linking to Anand's branch instead of the main apache beam version, this fixes that